apps/site/components/Common/Searchbox/Search.tsx

```diff
@@ -0,0 +1,227 @@
+'use client';
+
+import { SparklesIcon, DocumentTextIcon } from '@heroicons/react/24/outline';
+import {
+  MagnifyingGlassIcon,
+  ArrowTurnDownLeftIcon,
+  ArrowDownIcon,
+  ArrowUpIcon,
+} from '@heroicons/react/24/solid';
+import {
+  SearchInput,
+  FacetTabs,
+  SearchResults,
+  Suggestions,
+} from '@orama/ui/components';
+import { useSearchContext } from '@orama/ui/contexts';
+import classNames from 'classnames';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useLocale, useTranslations } from 'next-intl';
+import { type FC, type PropsWithChildren } from 'react';
+
+import styles from './search.module.css';
+import { getFormattedPath } from './utils';
+
+type SearchProps = PropsWithChildren & {
+  onChatTrigger: () => void;
+};
+
+export const Search: FC<SearchProps> = ({ onChatTrigger }) => {
+  const locale = useLocale();
+  const t = useTranslations();
+  const { searchTerm } = useSearchContext();
+
+  return (
+    <>
+      <SearchInput.Wrapper className={styles.searchInputWrapper}>
+        <MagnifyingGlassIcon />
+        <SearchInput.Input
+          inputId="doc-search"
+          ariaLabel={t('components.search.searchPlaceholder')}
+          placeholder={t('components.search.searchPlaceholder')}
+          className={styles.searchInput}
+        />
+      </SearchInput.Wrapper>
+
+      <div className={styles.chatButtonWrapper}>
+        <button
+          type="button"
+          onClick={onChatTrigger}
+          className={classNames(
+            styles.chatButton,
+            searchTerm ? styles.chatButtonWithSearch : ''
```

Comment:
```suggestion
            { [styles.chatButtonWithSearch]: searchTerm }
```

Ditto for the rest. `classNames` understands object arguments, so this statement can become:
```
classNames{ [styles.chatButton]: true, ... })
```